plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -500,7 +500,7 @@ object ScalafmtPlugin extends AutoPlugin {
     ScopeFilter(configurations = inAnyConfiguration)
 
   override def projectSettings: Seq[Def.Setting[_]] =
-    Seq(Compile, Test, IntegrationTest).flatMap(scalafmtConfigSettings) ++ Seq(
+    Seq(Compile, Test).flatMap(scalafmtConfigSettings) ++ Seq(
```

Comment:
can you please extract this change into a completely separate PR, with a link to explanation why this is deprecated/removed? that way, this important change would not be "hidden" in an otherwise innocuous "update to rc2" PR.

Reply:
Converted this into Draft. Will create PR for the change and then rebase this one.